src/bindings/python/src/compatibility/openvino/requirements-dev.txt

```diff
@@ -1,5 +1,7 @@
 opencv-python>=3.4.4.19
-pytest==4.0.1
-attrs==19.1.0
+pytest==4.0.1; python_version < '3.10'
+pytest==7.2.0; python_version >= '3.10'
+py>=1.9.0
```

Comment:
are you sure that `py` package requires to install separately? 
as I know pytest 7.2.0 removes dependency on it
https://github.com/pytest-dev/pytest/issues/10396

Reply:
We need to support a range of pytest versions due to multiple different Python versions used (and other repositories' dependencies). The reason it's added is there were CIs failing due to this package not being installed in the environment. Adding it helped with the issue.

Reply:
@eaidova BTW it is still open how to score this problem and other additional packages are still using `py` if you look at some comments like: https://github.com/pytest-dev/py/issues/287#issuecomment-1298933908 or here https://github.com/tox-dev/tox/issues/2524

It is still on-going effort to drop it everywhere and some of checks still shows the issue even when it is not present.
I think we should be dealing with it in separate issue - **not** here.

Reply:
I do not insist, just highlight potential problem which these changes bring

Reply:
However if `py` is a problem, project will face it eventually, even without this change. This package is already a dependency of others... probably a new task is coming.

Thanks for noticing it!